packages/form-builder/src/settings/donation-goal/index.js

```diff
@@ -19,18 +19,18 @@ const DonationGoalSettings = () => {
             value: 'amount-raised',
             label: __('Amount Raised', 'give'),
         },
-        {
-            value: 'percentage-raised',
-            label: __('Percentage Raised', 'give'),
-        },
-        {
-            value: 'number-donations',
-            label: __('Number of Donations', 'give'),
-        },
-        {
-            value: 'number-donors',
-            label: __('Number of Donors', 'give'),
-        },
+        // {
+        //     value: 'percentage-raised',
```

Comment:
I'm fine with commenting these out for now but let's add a comment about our intentions here

Reply:
I added a `@todo`, see https://github.com/impress-org/givewp-next-gen/pull/89/commits/2e4fa0c090dae97e3daac62a0909f84c80d7839f.